### PR TITLE
Add clickable docs url on error

### PR DESCRIPTION
**Pull request by fameowner99**

### CODE REVIEW

File: client/core/controllers/apiController.h [hunk 1]
```diff
@@ -20,7 +20,7 @@ public slots:
     void updateServerConfigFromApi(const QString &installationUuid, const int serverIndex, QJsonObject serverConfig);
 
 signals:
-    void errorOccurred(const QString &errorMessage);
+    void errorOccurred(ErrorCode errorCode);
```
Comment (Nethius) [inline client/core/controllers/apiController.h:23]: add const

File: client/ui/controllers/connectionController.cpp [hunk 2]
```diff
@@ -52,9 +51,9 @@ void ConnectionController::openConnection()
 
     if (serverConfig.value(config_key::configVersion).toInt()
         && !m_serversModel->data(serverIndex, ServersModel::Roles::HasInstalledContainers).toBool()) {
-        m_apiController.updateServerConfigFromApi(m_settings->getInstallationUuid(true), serverIndex, serverConfig);
+      m_apiController.updateServerConfigFromApi(m_settings->getInstallationUuid(true), serverIndex, serverConfig);
```
Comment (Nethius) [inline client/ui/controllers/connectionController.cpp]: incorrect tabulation

File: client/ui/controllers/connectionController.cpp [hunk 3]
```diff
@@ -52,9 +51,9 @@ void ConnectionController::openConnection()
...
     } else {
-        openConnection(false, serverConfig, serverIndex);
+      openConnection(false, serverConfig, serverIndex);
```
Comment (Nethius) [inline client/ui/controllers/connectionController.cpp]: incorrect tabulation

File: client/ui/controllers/connectionController.cpp [hunk 4]
```diff
@@ -40,8 +39,8 @@ void ConnectionController::openConnection()
 #if !defined(Q_OS_ANDROID) && !defined(Q_OS_IOS)
     if (!Utils::processIsRunning(Utils::executable(SERVICE_NAME, false), true))
     {
-        emit connectionErrorOccurred(errorString(ErrorCode::AmneziaServiceNotRunning));
-        return;
+      emit connectionErrorOccurred(ErrorCode::AmneziaServiceNotRunning);
```
Comment (Nethius) [inline client/ui/controllers/connectionController.cpp]: incorrect tabulation

File: client/ui/qml/Pages2/PageSetupWizardStart.qml [hunk 5]
```diff
@@ -76,9 +76,9 @@ PageType {
     Connections {
         target: InstallController
 
-        function onInstallationErrorOccurred(errorMessage) {
+        function onInstallationErrorOccurred(error) {
```
Comment (Nethius) [inline client/ui/qml/Pages2/PageSetupWizardStart.qml:79]: change all error to errorCode

**Review by Nethius — CHANGES_REQUESTED** [review]

File: client/core/controllers/apiController.h (hunk 1 above)
Comment (fameowner99) [reply client/core/controllers/apiController.h:23]: in private conversation decided that no sense to add here const
https://stackoverflow.com/questions/117293/does-using-const-on-function-parameters-have-any-effect-why-does-it-not-affect

**Review by Noxius87 — APPROVED** [review]

**Review by Nethius — APPROVED** [review]

**Review by lunardunno — APPROVED** [review]

**Review by pokamest — APPROVED** [review]

**Pull request merged**